Antidote/ContactCellView.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  ContactCellView.swift
+//  Antidote
+//
+//  Created by Yoba on 04/12/2016.
+//  Copyright © 2016 Dmytro Vorobiov. All rights reserved.
+//
+
+import Cocoa
+import SnapKit
+
+class ContactCellView: NSTableCellView {
+    var userAvatar: NSImageView!
+    var usernameTextField: NSTextField!
+    var lastMessageTextField: NSTextField!
```

Comment:
Also you can rename properties, it would make code more readable.

`usernameTextField` -> `usernameLabel`
`lastMessageTextField` -> `lastMessageLabel`